### PR TITLE
Users/v vbhanndari/remote save provider

### PR DESCRIPTION
**Pull request by v-vbhanndari**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -770,7 +770,7 @@
     "gulp-typescript": "^6.0.0-alpha.1",
     "mocha": "^9.2.2",
     "moment": "^2.29.4",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^2.6.8",
```
Comment (BidishaMS) [inline package.json:773]: What's the need of increasing the version?

File: package.json [hunk 2]
```diff
@@ -789,6 +789,7 @@
     "yargs": "^16.2.0"
   },
   "dependencies": {
+    "@types/node-fetch": "^2.6.2",
```
Comment (BidishaMS) [inline package.json:792]: Why the version different from devdependencies

File: package.json (hunk 1 above)
Comment (v-vbhanndari) [reply package.json:773]: I just ran npm install node-fetch and it took latest version

File: package.json (hunk 2 above)
Comment (v-vbhanndari) [reply package.json:792]: That is Type of so i think took latest by npm install

**Pull request merged**